setup.py

```diff
@@ -20,6 +20,7 @@
         "dev": ["pytest"],
         "spacy_predictors": ["spacy"],
         "lp_predictors": ["layoutparser", "torch", "torchvision", "effdet"],
+        "hf_predictors": ["torch", "transformers", "smashed@git+https://github.com/allenai/smashed@v0.0.1"],
```

Comment:
You can change this to `smashed==0.1.10` to install from PyPI instead.

Reply:
oku